slsim/Sources/SupernovaeCatalog/supernovae_sample.py

```diff
@@ -30,27 +33,55 @@ def supernovae_host_galaxy_offset(host_galaxy_catalog):
 
     offsets = []
     position_angle = []
+    e1 = []
```

Comment:
I think e1 and e2 should be return from this function and store them in host galaxy catalog. Otherwise, ellipticity component for each galaxy will be calculated in Galaxy class and it will not be consistent with e1 and e2 used here.

Reply:
Do you mean that the function should return both ra and dec offsets, as well as e1 and e2? Or should it be split into 2 different functions, one for calculating ra and dec offsets and one for calculating e1 and e2?

Reply:
I think function should return both ra, dec and e1, e2 so that you can use e1 and e2 in host galaxy catalog.

Reply:
Sounds great. That's what I chose to do in the latest commit!